mappings.py

```diff
@@ -0,0 +1,94 @@
+import os
+from pathlib import Path
+
+from constants import Dataset, PrintTemplate
+
+SUPPLIER_TO_SFTP_DIRECTORY = {
+    'QM': os.getenv('SFTP_QM_DIRECTORY'),
+    'PPO': os.getenv('SFTP_PPO_DIRECTORY')
+
+}
+
+SUPPLIER_TO_KEY_PATH = {
+    'QM': os.getenv('QM_PUBLIC_KEY_PATH') or str(
+        Path(__file__).parents[0].joinpath('dummy_keys', 'supplier_QM_dummy_public.asc')),
+    'PPO': os.getenv('PPO_PUBLIC_KEY_PATH') or str(
+        Path(__file__).parents[0].joinpath('dummy_keys', 'dummy_ppo_supplier_public_key.asc'))
+}
+
+SUPPLIER_TO_PRIVATE_KEY_PATH = {
+    'QM': 'dummy_keys/supplier_QM_dummy_private.asc',
+    'PPO': 'dummy_keys/dummy_ppo_supplier_private_key.asc'
```

Comment:
`dummy_ppo_supplier_public_key.asc` should be renamed to `supplier_PPO_dummy_private.asc`